services/provider/server/api_updater.go

```diff
@@ -0,0 +1,86 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+type APIUpdater interface {
```

Comment:
Let call it ConsomerAPI

Reply:
Actually why not call it provider like the protobuf?

Reply:
All these service related changes came from the [@sp98's PR](https://github.com/red-hat-storage/ocs-operator/pull/1430)